package.json

```diff
@@ -8,11 +8,12 @@
     "lint": "lerna run lint",
     "test": "lerna run test",
     "storybook": "lerna run storybook --parallel --stream",
-    "postinstall": "lerna run build --scope={vibe-storybook-components,@vibe/icons}",
-    "size": "lerna run size --scope=@vibe/core"
+    "postinstall": "lerna run build --scope={vibe-storybook-components,@vibe/icons}"
   },
   "devDependencies": {
+    "@size-limit/preset-small-lib": "^11.2.0",
```

Comment:
in the root? why not in @vibe/core?

Reply:
Since we're using it in a workflow only and not in the package specifically, and it can be used in further packages as well if we decide

Reply:
you can by the way install it only inside the workflow itself if it only gets used in the CI